pmm-app/tests/databaseChecks_test.js

```diff
@@ -1,78 +1,85 @@
 Feature('Database Failed Checks');
 const config = require('../pr.codecept.js').config.helpers.Playwright;
+const allure = codeceptjs.container.plugins('allure');
 const assert = require('assert');
 
-Before( async (I) => {
-    I.Authorize();
+Before(async (I) => {
+  I.Authorize();
 });
 
 Scenario(
     'PMM-T294 Verify user is able to see message about Disabled STT in Checks panel at Home Page @not-pr-pipeline',
     async (I, homePage, databaseChecksPage, settingsAPI) => {
-        await settingsAPI.apiDisableSTT();
-        I.amOnPage(homePage.url);
-        I.waitForVisible(homePage.fields.sttDisabledFailedChecksPanelSelector, 30);
-        I.see(databaseChecksPage.messages.homePagePanelMessage, homePage.fields.sttDisabledFailedChecksPanelSelector);
+      allure.severity('critical');
+      await settingsAPI.apiDisableSTT();
+      I.amOnPage(homePage.url);
+      I.waitForVisible(homePage.fields.sttDisabledFailedChecksPanelSelector, 30);
+      I.see(databaseChecksPage.messages.homePagePanelMessage, homePage.fields.sttDisabledFailedChecksPanelSelector);
     }
 );
 
-Scenario(
-    'PMM-T295 Verify user is able to see message about Disabled STT at Database Checks page @not-pr-pipeline',
+Scenario.only(
```

Comment:
Is this intentional? The other scenarios shouldn't run?
Better to leave a comment here.

Reply:
Oh yeah, that was committed accidentally, will remove it tomorrow